.github/workflows/build.yml

```diff
@@ -26,8 +26,6 @@ jobs:
           ./gradlew assembleRelease &&
           jar tvf $( find . -name '*-release.aar' ) |
           grep '\.so$'
-        env:
```

Comment:
I like to have fixed versions, else things start to break at random moments. 

So either change the `ubuntu:latest` to `ubuntu:2204` (or whatever), or re-add an NDK_VERSION here.

Reply:
the version is fixed in the build.cradle

since we need to pin it, I prefer to have it pinned explicitely in the relevant files instead of indirectly in the CI